docs/RELEASE.md

```diff
@@ -2,6 +2,12 @@
 
 Only `master` branch is used for releases.
 
+1.  Update _hadolint-docker_ entry version in the
+    [.pre-commit-hooks.yaml](../.pre-commit-hooks.yaml) file.
+    Use the new version you're going to release.
+    And push to master.
+
```

Comment:
This line causes the warning in remark-lint. There may only be single empty lines between list items.

Reply:
Thanks, I didn't notice it.